src/components/application_manager/test/state_controller/state_controller_test.cc

```diff
@@ -1728,8 +1799,8 @@ TEST_F(StateControllerImplTest, DISABLED_ActivateAppSuccessReceivedFromHMI) {
   using namespace hmi_apis;
   using namespace mobile_apis;
 
-  const uint32_t corr_id = 314;
-  const uint32_t hmi_app_id = 2718;
+  const uint32_t kCorrId = 314;
```

Comment:
@pvvasilev I saw that these const`s are used in a lot of test, probably it good place them in unnamed namespace ..